test/.motherhen-config.json

```diff
@@ -1,11 +1,57 @@
 {
-  "default": {
-    "vanilla": {
-      "tag": "central"
+  "formatVersion": "1.0.0",
+  "sources": {
+    "(default)": [
+      "hatchedegg"
+    ]
+  },
+  "patches": {
+    "(default)": {
+      "globs": [
+        "**/*.patch"
+      ],
+      "commitMode": "none",
+      "commitMessage": null
+    }
+  },
+  "integrations": {
+    "(default)": {
+      "vanillaTag": "release",
+      "sourceKey": "(default)",
+      "patchKey": "(default)",
+      "targetDirectory": "integrations/release"
+    },
+    "central": {
+      "vanillaTag": "central",
+      "sourceKey": "(default)",
+      "patchKey": "(default)",
+      "targetDirectory": "integrations/central"
+    }
+  },
+  "projects": {
+    "default": {
+      "integrationKey": "(default)",
+      "mozconfig": "optimized",
+      "appDir": "hatchedegg",
+      "displayAppName": "Hatched Egg (release)"
+    },
+    "central": {
+      "integrationKey": "central",
+      "mozconfig": "optimized",
+      "appDir": "hatchedegg",
+      "displayAppName": "Hatched Egg (central)"
+    }
+  },
+  "firefoxes": {
+    "optimized": {
+      "vanillaTag": "release",
```

Comment:
Is there a way for me to bind this to an upstream gecko version? e.g. `109.0`?

Reply:
Nevermind, it looks like I can use any of the unified tags, like `FIREFOX_109_0_1_RELEASE`.

Reply:
If you have to ask, then a README is a little bit unclear.  Not sure how to fix it though.

Reply:
Fixed